frontend/packages/kubevirt-plugin/src/utils/sort.ts

```diff
@@ -46,3 +47,77 @@ export const ignoreCaseSort = <T>(
   };
   return array.sort((a, b) => resolve(a).localeCompare(resolve(b)));
 };
+
+const getOSVersion = (osName: string): number[] =>
+  osName
+    .split(/\D/)
+    .filter((x) => x)
```

Comment:
this should test if it is a number

this would mean that `2.1.0` == `2.0.1`

Reply:
probably makes sense to do the map first

Reply:
@suomiy I think you are mistaken
`2.1.0` will yield `[2,1,0]`
`2.0.1` will yield `[2,0,1]`
The comparison afterwards will examine the two arrays by the value at the same index,
thus determine that `2.1.0` > `2.0.1`

Reply:
yes, you are right
I forgot that these are strings and not numbers.
```js
"0" == 0
// > true
!"0" == !0
// > false
```